docs/root/start/quick-start/run-envoy.rst

```diff
@@ -450,3 +450,14 @@ which are set to ``debug`` and ``trace`` respectively.
 .. tip::
 
    See ``ALL_LOGGER_IDS`` in :repo:`logger.h </source/common/common/logger.h#L29>` for a list of components.
+
+Reminder on docker networking
+-----------------------------
+
+In some environments, IPv6 is not supported by the docker networking. If you are running Envoy with any demo
+configurations, please set ``dns_lookup_family`` of cluster to the ``V4_ONLY`` just like following example.
+See `docker docs <https://docs.docker.com/config/daemon/ipv6/>`_ for more information about the IPv6 support.
```

Comment:
```suggestion
By default Envoy can use both IPv4 and IPv6 networks.

If your environment does not support ipv6 you should disable it.

This may be the case when using Docker on a non-linux host (see `here for more information regarding IPv6 support in Docker <https://docs.docker.com/config/daemon/ipv6/>`_.

You can disable IPv6 by setting the ``dns_lookup_family`` to ``V4_ONLY`` in your configuration as follows:

```